index.html

```diff
@@ -12,7 +12,7 @@
         <link rel="stylesheet" href="./styles/styles.css">
         <title>Home</title>
     </head>
-    <body class="flex-column">
+    <body class="contact-container">
```

Comment:
If index.html is going to have this class on body, it shouldn't be called contact-container

Was the change to index.html not intentional in this branch?

Reply:
it was a mistake, i thought i had removed it but i guess notor maybe i still dont know how to use git well.